api/src/main/java/org/openmrs/module/eptsreports/reporting/library/queries/UsMonthlySummaryQueries.java

```diff
@@ -0,0 +1,291 @@
+        + "                       ON p.patient_id = e.patient_id  \n"
+        + "               inner join obs o  \n"
+        + "                       ON o.encounter_id = e.encounter_id  \n"
+        + "        WHERE  e.voided = 0  \n"
+        + "               AND o.voided = 0  \n"
+        + "               AND p.voided = 0  \n"
+        + "               AND e.encounter_type IN ( 18, 6, 9 )  \n"
+        + "               AND o.concept_id = 1255  \n"
+        + "               AND o.value_coded = 1256  \n"
+        + "               AND e.encounter_datetime <= :endDate  \n"
+        + "               AND e.location_id = :location  \n"
+        + "        GROUP  BY p.patient_id  \n"
+        + "        UNION  \n"
+        + "        -- leva a primeira ocorrencia do conceito 1190: Data de Inicio de TARV  \n"
+        + "        SELECT p.patient_id,  \n"
+        + "               Min(value_datetime) data_inicio  \n"
+        + "        FROM   patient p  \n"
+        + "               inner join encounter e  \n"
+        + "                       ON p.patient_id = e.patient_id  \n"
+        + "               inner join obs o  \n"
+        + "                       ON e.encounter_id = o.encounter_id  \n"
+        + "        WHERE  p.voided = 0  \n"
+        + "               AND e.voided = 0  \n"
+        + "               AND o.voided = 0  \n"
+        + "               AND e.encounter_type IN ( 18, 6, 9 )  \n"
+        + "               AND o.concept_id = 1190  \n"
+        + "               AND o.value_datetime IS NOT NULL  \n"
+        + "               AND o.value_datetime <= :endDate  \n"
+        + "               AND e.location_id = :location  \n"
+        + "        GROUP  BY p.patient_id  \n"
+        + "        UNION  \n"
+        + "        -- leva a primeira ocorrencia da inscricao do paciente no programa de Tratamento ARV  \n"
+        + "        SELECT pg.patient_id,  \n"
+        + "               date_enrolled data_inicio  \n"
+        + "        FROM   patient p  \n"
+        + "               inner join patient_program pg  \n"
+        + "                       ON p.patient_id = pg.patient_id  \n"
+        + "        WHERE  pg.voided = 0  \n"
+        + "               AND p.voided = 0  \n"
+        + "               AND program_id = 2  \n"
+        + "               AND date_enrolled <= :endDate  \n"
+        + "               AND pg.location_id = :location  \n"
+        + "         UNION  \n"
+        + "         -- Leva a data do primeiro levantamento de ARV para cada paciente: Data do primeiro Fila do paciente \n"
+        + "         SELECT e.patient_id,  \n"
+        + "                Min(e.encounter_datetime) AS data_inicio  \n"
+        + "         FROM   patient p  \n"
+        + "                inner join encounter e  \n"
+        + "                        ON p.patient_id = e.patient_id  \n"
+        + "         WHERE  p.voided = 0  \n"
+        + "                AND e.encounter_type = 18  \n"
```

Comment:
Like this encounter type, can it be accessed from the hivmetadata file?

Reply:
sure, thanks, i just made some response, i still can do this but am evaluating how worth is this!